packages/mds-provider-service/repository/index.ts

```diff
@@ -53,11 +53,17 @@ export const ProviderRepository = ReadWriteRepository.Create(
       }
     }
 
-    const getProviders = async (): Promise<ProviderDomainModel[]> => {
+    const getProviders = async (options: GetProvidersOptions): Promise<ProviderDomainModel[]> => {
       try {
+        const { provider_types } = options ?? {}
```

Comment:
Wouldn't this make the default behavior to match (and return) no providers at all?

Reply:
No. The base of the query is a `select * from providers` and if the `provider_types` are present, then a `where` clause will be appended to narrow things down.

Reply:
Right, ok, if provider_types is undefined then that if is skipped. 👍